Phonebook.Frontend/src/app/shared/dialogs/user-information/incorrect-user-information.component.html

```diff
@@ -0,0 +1,34 @@
+<div class="">
+  <h2 mat-dialog-title i18n="User-InformationDialog|Headline@@Headline">Incorrect User Information</h2>
+  <div mat-dialog-content *ngIf="onMyProfile">
+    <span i18n="User-InformationDialog|Profile@@Your Profile">
+      Dear {{ data.person.Firstname }},<br />
+      unfortunately, you can't change your user information via the Phonebook interface. <br />
+      If your data is incorrect, please contact your Human Resources department.
+    </span>
+  </div>
+  <div mat-dialog-content *ngIf="!onMyProfile">
+    <span i18n="User-InformationDialog|Profile@@Colleague Profile">
+      Is there something wrong with the profile? <br />
+      Please let your colleague know that the data is incorrect and that it can be changed through your Human Resources
+      department.
+    </span>
+  </div>
+  <div *ngIf="!onMyProfile">
+    <a
+      mat-icon-button
+      (click)="sendMail()"
+      i18n-matTooltip="User-InformationDialog|Button@@SendEmail"
+      matTooltip="Send your colleague a direct message"
+      i18n="User-InformationDialog|Profile@@Notify"
+    >
+      <mat-icon mat-list-icon>email</mat-icon>
+      Notify {{ data.person.Firstname }}
+    </a>
+  </div>
+  <div align="end">
+    <button mat-button mat-dialog-close cdkFocusInitial i18n="User-InformationDialog|Button@@Button">
```

Comment:
Please use a different (more specific) id for the button, as I think somebody might give it the same id. If you want to make a general translation for a button you can either use https://t-systems-mms.github.io/phonebook/pages/development-guides/frontend/translation-guide.html#triangular_flag_on_post-predefined-message-ids these or add you translation to it.